scooby/report.py

```diff
@@ -126,6 +127,44 @@ def packages(self):
         return pckg_dict
 
 
+    def listing(self):
```

Comment:
Add a parameter `write=True` or similar. If `True` (default), the result is written to `requirements.txt`. If `False`, it is returned to the user.